code/game/objects/items/blueprints.dm

```diff
@@ -470,11 +470,12 @@ these cannot rename rooms that are in by default BUT can rename rooms that are c
 	icon = 'icons/obj/items.dmi'
 	icon_state = "blueprints"
 	desc = "Required for turning a dull room with some engines in the back into something that can move through space!"
+	var/adminbus = FALSE //VV to 1 for !!FUN!!
```

Comment:
retarded comment, explain what it does and/or give it a better name

Reply:
i couldn't come up with any better names, if you have an idea let me know

Reply:
`can_be_shuttle`?

Reply:
Then we could also drop the type check and just set that var to `TRUE` for custom areas.

Reply:
Do the thing 🅱️ lease

Reply:
having to mass-edit an area seems like it'd be more hassle than just overriding the typecheck on the licence, if you think otherwise i'll try to get it set up for areas instead